apps/typed-dependency-tester/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "typed-dependency-tester",
+  "private": true,
+  "author": "Microsoft Teams",
+  "description": "A tester to check if types in the dependency array will cause errors",
+  "version": "0.0.1",
+  "scripts": {
+    "build": "pnpm i && pnpm copy && pnpm tsc && pnpm clean",
+    "copy": "cp ./node_modules/@microsoft/teams-js/dist/MicrosoftTeams.d.ts ./ || xcopy .\\node_modules\\@microsoft\\teams-js\\dist\\MicrosoftTeams.d.ts .\\ /Y",
```

Comment:
I've never seen this syntax, that's cool! Am I correct in interpreting this as it will attempt to use `cp` first and if that fails, it will use `xcopy`?

Reply:
Yes this is exactly the case, if cp fails it will use xcopy instead. For example, an OS compatibility issue.